scripts/file-format-lint.py

```diff
@@ -0,0 +1,100 @@
+import re, sys, os, codecs
```

Comment:
Suggest maintaining this source code in your own repo and reference it in the circleci config file, so for further code changes, you don't have to open pull requests in 5 branches each time.